### PR TITLE
Fix missing resources needed for ament

### PR DESCRIPTION
**Pull request by pbaughman**
- See https://github.com/colcon/colcon-ros/pull/74 and https://discourse.ros.org/t/can-we-chat-about-breaking-changes-in-stable-releases/10689 for background

See the same issue fixed in the 'launch' package about three weeks ago: https://github.com/ros2/launch/pull/323

Signed-off-by: Pete Baughman <pete.baughman@apex.ai>

### CODE REVIEW

**Comment by pbaughman** [comment]
@ruffsl This is blocking https://github.com/ros2/sros2/pull/158 - can we find a way to approve/merge this relatively quickly?

Thanks!

**Comment by ruffsl** [comment]
Could you rebase off of master? I think part of this already exists from https://github.com/ros2/sros2/pull/159

File: sros2/setup.py [hunk 1]
```diff
@@ -64,4 +66,8 @@ def package_files(directory):
     package_data={
         'sros2': extra_files,
     },
+    data_files=[
+        ('share/' + package_name, ['package.xml']),
+        ('share/ament_index/resource_index/packages', ['resource/'+package_name]),
```
Comment (ruffsl) [inline sros2/setup.py]:
```suggestion
        ('share/ament_index/resource_index/packages', ['resource/' + package_name]),
```

**Comment by pbaughman** [comment]
@ruffsl shoot, you're right - I think I pulled from my fork since that's where I've been pushing changes, not from sros2

**Comment by pbaughman** [comment]
@ruffsl done

**Comment by pbaughman** [comment]
@ruffsl You will have to push your change again, but commit with the '-s' option so the commit is signed off

**Comment by ruffsl** [comment]
Yea, I made the commit via the github UI and it doesn't seem possible to manually sign off a commit from the web. I also can't figure out how to push to Github PR branch I've fetched, as fetching doesn't set the remote target, I get a stale info, or git creates a new branch of the same name on origin:  
https://help.github.com/en/articles/committing-changes-to-a-pull-request-branch-created-from-a-fork

Let me know if you know the command to push onto a forked PR, or feel free to force push to own my change.

**Comment by pbaughman** [comment]
@ruffsl That's annoying - I have force pushed a commit with the appropriate sign-offs

**Pull request merged**